ui/main_window.cpp

```diff
@@ -3301,17 +3301,24 @@ CorrelationTarget            target)
         return std::nullopt;
     }
 
-    uint64_t       node_index = (uint64_t)(source_index.internalPointer());
-    Dive::NodeType node_type = m_data_core->GetCommandHierarchy().GetNodeType(node_index);
-    bool           type_is_valid = false;
+    uint64_t                      node_index = (uint64_t)(source_index.internalPointer());
```

Comment:
static_cast just to be consistent with other code :)

Reply:
more for consistent with using C++ style than C style. i dont really have a strong opinion on this though, your call :)

Reply:
This should be `internalId()`

I don't know why are we using `internalPointer()`